core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -40,8 +41,8 @@ import org.junit.Assert._
 import org.junit.Test
 import org.scalatest.Assertions.intercept
 
+import scala.collection.JavaConverters._
 import scala.collection.{Seq, mutable}
-import scala.jdk.CollectionConverters._
 
 class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
```

Comment:
Would it make sense to also add a test which exercice the whole, from the command line to the admin client? I think that we already have a couple of such tests.

Reply:
Hmm. I'm not finding those for `kafka-configs.sh`. Can you point me at where they are?

Reply:
Sure. I was thinking about something like this one: https://github.com/apache/kafka/blob/779c8fae389fb3052348ee5130b2aedffe96023b/core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala#L486

Reply:
Ah, thanks, I thought you meant an integration test using an actual admin client. I've added code to use the mock admin client with the file option. Thanks for the suggestion!